java-vtl-parser/src/main/antlr4/no/ssb/vtl/parser/VTL.g4

```diff
@@ -169,6 +170,13 @@ ISNOTNULL : 'is not null' ;
 
 nvlFunction : 'nvl' LPAR expression COMMA expression RPAR ;
 
+ifThenElseExpression: 'if' ifBody ('elseif' ifBody)* ('else' ifBodyExpression) ;
+
+ifBody : ifBodyExpression 'then' ifBodyExpression ;
+
+ifBodyExpression : ifThenElseExpression {notifyErrorListeners("cannot embed if");}
```

Comment:
this message is a bit hard to understand.

Reply:
OK, what do you propose?

Reply:
imbricated if-then-else expressions
or
value cannot be another if-then-else expression
?

Reply:
The latter sounds good :)